src/ui/components/PlayerPicture.tsx

```diff
@@ -20,7 +20,7 @@ const PlayerPicture = ({
 }) => {
 	const [wrapper, setWrapper] = useState<HTMLDivElement | null>(null);
 	useEffect(() => {
-		if (face && !imgURL && wrapper) {
+		if (face && (!imgURL || imgURL == "/img/blank-face.png") && wrapper) {
```

Comment:
I think this change (treat the blank-face URL as special, and put it behind a cartoon face in priority) could be confusing. I would prefer keeping the old behavior - imgURL always has priority. You can set imgURL to an empty string (ugh why did I make that the "undefined" value rather than just using `undefined`) when applying the face objects.

(I think ideally I only would have supported either imgURL or face being present, probably by putting them on the same property of a player object, similar to how they're treated by this PR in the global data. But it's not worth a schema change to fix that now.)